mobile/asa-go/src/components/map/mapOperations.ts

```diff
@@ -0,0 +1,26 @@
+import { fireZoneExtentsMap } from "@/fireZoneUnitExtents";
+import { Position } from "@capacitor/geolocation";
+import { fromLonLat } from "ol/proj";
+
+export const findFireZoneForLocation = (userPosition: Position) => {
```

Comment:
Maybe hold on to this in case we need it in the future but I would pull it out of this PR since it actually isn't used. I know it is old code but I'm not a fan of the hard coded extents and source ids we have in place in `fireZoneExtentsMap`. There's a good chance our ids will end up out of sync between the db, the pmtiles layer and the extents map.